package.json

```diff
@@ -25,10 +25,17 @@
     "@babel/preset-react": "^7.14.5",
     "babel-loader": "^8.2.2",
     "babel-plugin-transform-class-properties": "^6.24.1",
+    "css-loader": "^6.5.1",
+    "style-loader": "^3.3.1",
     "webpack-cli": "^4.9.0"
   },
   "dependencies": {
+    "@emotion/react": "^11.6.0",
+    "@emotion/styled": "^11.6.0",
+    "@mui/icons-material": "^5.2.0",
+    "@mui/material": "^5.2.1",
     "axios": "^0.23.0",
+    "bootstrap-icons": "^1.7.1",
```

Comment:
what are all these imports for? react icons is already being used, which uses fa bootstrap icons. can you please use that?